plugins/obs-outputs/librtmp/rtmp.c

```diff
@@ -1561,16 +1557,6 @@ WriteN(RTMP *r, const char *buffer, int n)
 #ifdef CRYPTO
     char *encrypted = 0;
     char buf[RTMP_BUFFER_CACHE_SIZE];
```

Comment:
It looks like these variables (and the whole `#ifdef CRYPTO`) can also be removed, along with the code at the end of the function.

https://github.com/tytan652/obs-studio/blob/447366d9e134537cf3c37ba8cab56298ef436abe/plugins/obs-outputs/librtmp/rtmp.c#L1597-L1600

Reply:
Changes pushed